micro-deps/micro-deps/src/test/groovy/com/ofg/infrastructure/discovery/ServiceConfigurationResolverSpec.groovy

```diff
@@ -53,4 +54,19 @@ class ServiceConfigurationResolverSpec extends Specification {
         then:
             resolver.dependencies == [:]
     }
+
+    def 'should provide load balancer type for given service path'() {
+        given:
+            def resolver = new ServiceConfigurationResolver(LOAD_BALANCING_DEPENDENCIES)
+        expect:
+            resolver.getLoadBalancerTypeOf('com/ofg/ping') == ProviderStrategyFactory.STICKY
```

Comment:
This can be turned into data table with `@Unroll`, is it worth it for two options? Or maybe you can then add more just for fun?